swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/model/Resource.kt

```diff
@@ -74,6 +78,20 @@ abstract class Resource : Excludable, Timestamped, HasDetails() {
   override fun toString(): String {
     return details.toString()
   }
+
+  fun getTagValue(key: String): String? {
```

Comment:
appversion (what we parse for the package name for images) is stored as an amazon tag on an image